src/ontology/bridge/aio-bridge-to-upper.tsv

```diff
@@ -0,0 +1,9 @@
+CJM col 1	CJM col 2	AIO current label	AIO propsed label	OBO label	OBO id
+		ID			SC % 
+Bias	disposition	aio:Bias		disposition	BFO:0000016
+Function	rename to MathematicalFunction, place under Planned Process?	aio:Function	aio:MathematicalFunction	planned process	OBI:0000011
+Layer	information artefavt	aio:Layer		information content entity	IAO:0000030
```

Comment:
typo + different case:

information artefavt

Reply:
thanks that column has been removed